### ISSUE TITLE
First project

### ISSUE THREAD

**Issue by Serg-dev22** [zaggino/brackets-git#916]
[I'm supposed to describe steps to replicate my issue here and if I don't do that, I understand that my issue will be probably ignored.]

--------------------------------

*Brackets 1.1.0-15558 (linux), Git 1.9.1 & Brackets-Git 0.14.11*

```
#1. TypeError: Object #<Object> has no method 'triggerHandler'
```

**Comment by nacholozano** [comment]
I had the same problem. I saw this problem when I tried init a repository from brackets.
This worked for me: 
1 - Create a repository in guthub
2 - Commit changes
3 - Pull the repository
4 - Push the changes

**Comment by zaggino** [comment]
Fixed in `0.14.13`, please confirm.
